src/ipc/ipc4/handler.c

```diff
@@ -38,8 +38,15 @@
 struct ipc4_msg_data {
 	uint32_t msg_in[2]; /* local copy of current message from host header */
 	uint32_t msg_out[2]; /* local copy of current message to host header */
+	bool delay_reply;
+	uint32_t delay_error;
 };
 
+static struct ipc4_msg_data msg_data;
```

Comment:
SHARED ?

Reply:
sorry, I don't get your point " SHARED". Thanks!

Reply:
I means this needs to be shared data (across DSP cores) - use the SHARED prefix.

Reply:
thanks!